packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/CompareModelsConceptsResponseHandler.java

```diff
@@ -0,0 +1,40 @@
+package software.uncharted.terarium.hmiserver.service.tasks;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.List;
+import java.util.UUID;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class CompareModelsConceptsResponseHandler extends TaskResponseHandler {
+
+	public static final String NAME = "mira_task:model_comparison";
+
+	@Override
+	public String getName() {
+		return NAME;
+	}
+
+	@Data
+	public static class Input {
+
+		List<String> amrs;
+	}
+
+	@Data
+	public static class Properties {
+
+		List<UUID> modelIds;
+		UUID workflowId;
+		UUID nodeId;
+	}
```

Comment:
The abstractions level here feels a little strange: why are the input attributes of a taskrunner-request part of the output/response handler?  But we can sort these out later, just want to make a note.

Reply:
It's the opposite actually.
Because the task response can be consumed in another place, we can provide extra information as part of the request to be provided in the output.
In this case, this allow the notiifcationGroup to provide a link to the right drilldown to open.